pkgs/top-level/python-packages.nix

```diff
@@ -2311,6 +2311,22 @@ in {
 
   humanfriendly = callPackage ../development/python-modules/humanfriendly { };
 
+  verboselogs = callPackage ../development/python-modules/verboselogs { };
+
+  naturalsort = callPackage ../development/python-modules/naturalsort { };
+
+  property-manager = callPackage ../development/python-modules/property-manager { };
+
+  capturer = callPackage ../development/python-modules/capturer { };
+
+  update-dotdee = callPackage ../development/python-modules/update-dotdee { };
+
+  executor = callPackage ../development/python-modules/executor { };
+
+  coloredlogs = callPackage ../development/python-modules/coloredlogs { };
+
+  rotate-backups = callPackage ../tools/backup/rotate-backups { };
```

Comment:
I personally would like to see this dispersed in some attempt to alphabetize them among the other packages.

Reply:
Tried, but it is just one big chaos. Can't find where `r` starts or ends at all

Reply:
i agree in general as well, that's why i said attempted :) usually i search for "  r" which will give me the first entry which begin with r as a package definition, then just go from there.